rules/empty-brace-spaces.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+
+const MESSAGE_ID = 'empty-brace-spaces';
+const messages = {
+	[MESSAGE_ID]: 'Do not add spaces between braces.'
+};
+
+const selector = `:matches(${
+	[
+		'BlockStatement[body.length=0]',
+		'ClassBody[body.length=0]',
+		'ObjectExpression[properties.length=0]'
+	].join(', ')
+})`;
+
+const create = context => {
+	const sourceCode = context.getSourceCode();
+	return {
+		[selector](node) {
+			let [start, end] = node.range;
```

Comment:
We should not use the unofficial properties `start` and `end`.

Reply:
That's destructuring `.range`  :smile:

Reply:
Oh... I missed that.